lib/bitcrowd_ecto/changeset.ex

```diff
@@ -296,8 +297,8 @@ defmodule BitcrowdEcto.Changeset do
   Validates a date field in the changeset is after the given reference date.
   """
   @doc since: "0.11.0"
-  @spec validate_date_after(Ecto.Changeset.t(), atom, Date.t(), [{:formatter, fun}]) ::
-          Ecto.Changeset.t()
+  @spec validate_date_after(Changeset.t(), atom, Date.t(), [{:formatter, fun}]) ::
```

Comment:
nitpick: I'd prefer if we kept refactorings / renames like this separate from the functional changes, i.e. a simple PR afterwards, to make reviews easier. Don't care as much about this in other contexts, but for OSS libs I feel we should keep code quality even higher than usual, and also try to set examples with what we do, so that others can be inspired by the great things already there :)

no need to change anything though, just my 2cts